en/intro_to_command_line/README.md

```diff
@@ -185,14 +185,30 @@ Music
 
 Now, let's go to our Desktop directory:
 
-<!--sec data-title="Change current directory: OS X and Linux" data-id="OSX_Linux_move_to" data-collapse=true ces-->
+<!--sec data-title="Change directory: OS X" data-id="python_OSX"
```

Comment:
Was this separation into two lines intentional?

Reply:
Do you mean the separation in OS-X-specific and Linux-specific expandable sections instead of a common expandable section for OS X _and_ Linux? I've adopted that from [the original remark](https://github.com/DjangoGirls/tutorial/pull/1465/files#diff-0ce35d3afd321211a71112aa5f36d20aL591) which came in a later chapter and which is also being changed by this PR. And yes, adopting it was intentional, as I don't know whether OS X also has localized home default subfolder names.

Or do you mean the line break within the HTML comment (the GitBook pragma starting the expandable section)? That was also adopted from the other chapter, but unintentionally so. Does it matter? I can change it to a single line if wanted.

Reply:
I meant within the HTML comment. I don't care one way or the other, but it seems unrelated to the rest of this PR?